awx/api/serializers.py

```diff
@@ -3438,6 +3438,12 @@ def get_related(self, obj):
             res['organization'] = self.reverse('api:organization_detail',   kwargs={'pk': obj.organization.pk})
         if obj.webhook_credential_id:
             res['webhook_credential'] = self.reverse('api:credential_detail', kwargs={'pk': obj.webhook_credential_id})
+        if obj.inventory:
+            res['inventory'] = self.reverse(
+                'api:inventory_detail', kwargs={
+                    'pk': obj.inventory.pk
```

Comment:
could be more efficient to consistently use `obj.inventory_id` kind of syntax. It's the pattern I would generally prefer, not a blocker for this specifically, because current code is highly inconsistent.

Reply:
Fair enough, I had that thought as well.